composer.lock

```diff
@@ -4,7 +4,11 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
+<<<<<<< HEAD
     "content-hash": "9dc0b30676fe8bb9b0dd9631179d7d7b",
+=======
+    "content-hash": "e42f3095c737c9b2d79bd4ca048582bb",
+>>>>>>> 258f271 (Add support to php 8.0)
```

Comment:
There is a whole bunch of them across the file. My suggestion would be to remove the lock file and run `composer update -W`